enhancements/network/infiniband.md

```diff
@@ -0,0 +1,123 @@
+---
+title: infiniband
+authors:
+  - "@zshi"
+reviewers:
+  - "@pliurh"
+  - "@dougbtv"
+  - "@s1061123"
+  - "@danwinship"
+  - "@squeed"
+  - "@dcbw"
+  - "@yrobla"
+approvers:
+  - "@knobunc"
+  - "@fepan"
+creation-date: 2020-05-06
+last-updated: 2020-05-06
+status: implementable
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Infiniband
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+RDMA (Remote Direct Memory Access) fabrics such as Infiniband and RDMA over
+Converged Ethernet (RoCE) are key to unlocking scalable performance for
+demanding Artificial Intelligence (AI) and Machine Learning (ML) applications.
+RoCE v2 is supported in SR-IOV Operator as Tech Preview since 4.3, This
+enhancement is to introduce native Infiniband support.
+
+## Motivation
+
+Enable high-performance message transmission via RDMA for distributed workloads
+such as AI/ML.
+
+### Goals
+
+- Provide native Infiniband support on pod additional network interfaces
+
+### Non-Goals
+
+- Support RoCE v2
```

Comment:
So from the Summary you already support it in SR-IOV Operator, right?

Reply:
Yes, RoCE v2 on selected NICs as Tech Preview since 4.3

Reply:
RoCE is ethernet udp traffic , i think that's no really related to this doc right ?

Reply:
yes, it's listed as a non-goal

Reply:
removed